cmd/clusterctl/client/repository/components.go

```diff
@@ -57,7 +57,7 @@ const (
 // 3. Ensure all the ClusterRoleBinding which are referencing namespaced objects have the name prefixed with the namespace name
 // 4. Adds labels to all the components in order to allow easy identification of the provider objects.
 type Components interface {
-	// configuration of the provider the provider components belongs to.
+	// Provider holds configuration of the provider components belongs to.
```

Comment:
```suggestion
	// Provider holds configuration of the provider the provider components belong to.
```

I think the "name of the field" in case of interfaces is not Provider the functions will be just inlined, but fix looks good anyway +/- I think it broke the sentence :)

Reply:
Should be fixed now 😃